lib/__testfixtures__/replace-constructors.input.js

```diff
@@ -1,3 +1,9 @@
 const n = new Number();
 const t = new TypeError();
 const s = new Set([1, 2, 3]);
+class NotGlobal {
```

Comment:
What is this added for? Does it fail before these changes?

Reply:
Yeah I realized that it was a necessary test while refactoring and just threw it in, but it probably ought to go in a different PR